src/java/tagpost/proto/tagpost.proto

```diff
@@ -13,10 +13,13 @@ service Response {
 
 // request message 
 message Request {
-  string noticekey = 1;
+  string notice_key = 1;
+  string id = 2;
```

Comment:
Please be specific about this ID. Is it request ID or some other IDs?

Reply:
Fixed, changed to request_id